--- a/shell.c	2022-07-21 16:02:15.000000000 +0000
+++ b/shell.c	2022-07-27 17:11:39.818601121 +0000
@@ -23071,6 +23071,8 @@
 #  define main fiddle_main
 #endif
 
+extern void init_mvsqlite();
+
 #if SQLITE_SHELL_IS_UTF8
 int SQLITE_CDECL main(int argc, char **argv){
 #else
@@ -23353,6 +23355,7 @@
     }
   }
 
+  init_mvsqlite();
   if( data.pAuxDb->zDbFilename==0 ){
 #ifndef SQLITE_OMIT_MEMORYDB
     data.pAuxDb->zDbFilename = ":memory:";
